modules/tensor_mechanics/src/materials/TemperatureDependentHardeningStressUpdate.C

```diff
@@ -80,22 +84,26 @@ TemperatureDependentHardeningStressUpdate::TemperatureDependentHardeningStressUp
       new LinearInterpolation(_hf_temperatures, yield_stress_vec));
```

Comment:
Should probably be a unique pointer with `std::make_unique<LinearInterpolation>(_hf_temperatures, yield_stress_vec);` here